### ISSUE TITLE
when searching using "/" in NORMAL mode, I can not use "n" or "N" to go between search results.

### ISSUE THREAD

**Issue by epiq-dhelm** [VSCodeVim/Vim#8916]
**Describe the bug**
when searching using "/" in NORMAL mode, I can not use "n" or "N" to go between search results.
But if I use "?" to search backwards, the "n" and "N" work.
When I do "/" it will find the next search word, but then the "n" and "N" does not work.  Nothing happens when I hit n or N.
If I have previously done a backwards search ?<word1> then do the /<word2> search, it will find the next word2, then if I hit "n" it will got backwords to word1.

**To Reproduce**
Ever since I installed the vim extension in my vscode for Linux it has not worked.

**Environment (please complete the following information):**

- Extension (VsCodeVim) version:Vim 1.27.2
- VSCode version:
Version: 1.86.0
Commit: 05047486b6df5eb8d44b2ecd70ea3bdf775fd937
Date: 2024-01-31T10:27:46.147Z
Electron: 27.2.3
ElectronBuildId: 26495564
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Linux x64 5.14.0-1051-oem
- OS:
Ubuntu 20.04

**Comment by raihua** [comment]
Hi epiq,
I had a look at this and it seems to work if you press enter after typing /searchterm. N and n work for me that way. Can you please let me know if that works for you?

**Comment by epiq-dhelm** [comment]
That is the issue.  It is NOT working for me.  Not sure how to debug it,
and I can't use it if that doesn't work.

**Comment by heron-he** [comment]
Hello, have you solved it yet? I also have this issue

**Comment by epiq-dhelm** [comment]
Yeah, I typically do :/ before my search term, and that did not work.  

If I just do /searchterm it works on one of my computers and not the other.  Trying to solve in the background.
